packages/itwinui-css/src/searchbox/searchbox.scss

```diff
@@ -5,11 +5,21 @@
   inline-size: fit-content;
   padding-inline: 0; // To remove 1px padding from input-flex-container
 
-  // Expanded
   &:where([data-iui-expanded='true']) {
     inline-size: 100%;
     border-color: var(--_iui-input-border-color);
   }
+
+  &:not([data-iui-expanded='true']) {
+    &:is(:hover, :not(:hover)) {
```

Comment:
This selector shouldn't be necessary I think? Are we not able to just use `background-color: transparent` directly?